### ISSUE TITLE
dev.Dockerfile line 20 has a hardcoded meteor version.

### ISSUE THREAD

**Issue by ViddeM** [cthit/EatIT#46]

**Comment by lindskogen** [comment]
What is the issue here? 😃

**Comment by ViddeM** [comment]
The problem seems to be that docker tries to use filepath to docker which includes "1.6.0" when the meteor version downloaded in a previous step is 1.6.1.

**Comment by lindskogen** [comment]
I see! After some research I found out that you can give a `release` parameter: https://install.meteor.com/?release=1.6.0

We can use a variable here:

```
ENV METEOR_VERSION 1.6.1


RUN curl https://install.meteor.com/?release=$METEOR_VERSION | sh
USER root:root
RUN cp /home/meteor/.meteor/packages/meteor-tool/$METEOR_VERSION/mt-os.linux.x86_64/scripts/admin/launch-meteor /usr/bin/meteor
```

**Comment by hulthe** [comment]
Solved by #47
